src/parse_instruction.py

```diff
@@ -336,10 +361,11 @@ def parse_instruction(line: str, meta: InstructionMeta, arch: ArchAsm) -> Instru
         # Parse arguments.
         args: List[Argument] = list(
             filter(
-                None, [parse_arg(arg_str.strip()) for arg_str in args_str.split(",")]
+                None,
+                [parse_arg(arg_str.strip(), arch) for arg_str in args_str.split(",")],
             )
         )
         instr = Instruction(mnemonic, args, meta)
         return arch.normalize_instruction(instr)
-    except Exception as e:
+    except:
```

Comment:
```suggestion
    except Exception:
```

Reply:
Is this to make it explicit that we're handling all exceptions? Whatever it is, I can change it.

Reply:
Bare `except` is dangerous: https://www.flake8rules.com/rules/E722.html